ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java

```diff
@@ -0,0 +1,46 @@
+package io.syndesis.qe.steps.other;
+
+import static org.junit.Assert.assertEquals;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Selenide;
+
+import java.util.HashMap;
+
+import cucumber.api.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DocsVerificationSteps {
+    private HashMap<String, String> versions = new HashMap<>();
+    private static String versionOnAboutPage;
+
+    public DocsVerificationSteps() {
+        versions.put("1.9-SNAPSHOT", "7.5");
+    }
+
+    @Then("^check version in about page$")
+    public void checkVersion() {
+        final By SYNDESIS_VERSION = By.cssSelector("[data-testid=\"about-modal-content-version-list-item\"]");
+        assertThat($(SYNDESIS_VERSION).shouldBe(visible).getText())
+            .isNotEmpty();
+        versionOnAboutPage = $(SYNDESIS_VERSION).getText();
+        assertThat(versions.keySet()).contains(versionOnAboutPage);
+    }
+
+    @Then("verify whether the docs has right version")
+    public void verifyDocsVersion() {
+        Selenide.switchTo().window(1);
+        final By DOCS_VERSION = By.cssSelector(".productnumber");
+        assertThat($(DOCS_VERSION).shouldBe(visible).getText())
+            .isNotEmpty();
+        String docsVersion = $(DOCS_VERSION).getText();
+        assertEquals(versions.get(versionOnAboutPage), docsVersion);
```

Comment:
use `assertThat` here as well

Reply:
also you are relying on the fact that the test before will be executed - the test shouldn't be depending on any other tests and should be "standalone" - so that when you run only one selected test, it will pass